src/test/java/org/jpeek/web/DyNumTest.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2019 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.jpeek.web;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Test case for {@link DyNum}.
+ * @author Nikita Puzankov (humb1t@yandex.ru)
+ * @version $Id$
+ * @since 0.31
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class DyNumTest {
+
+    @Test
+    public void testConstructDyNumFromString() {
+        final double number = 2.0f;
+        final DyNum dynum = new DyNum(
+            String.valueOf(number)
+        );
+        MatcherAssert.assertThat(
+            dynum.doubleValue(),
+            Matchers.equalTo(number)
+        );
+    }
+
+    @Test
+    public void testConstructDyNumFromLong() {
+        final Long number = Long.valueOf(2);
+        final DyNum dynum = new DyNum(number);
+        MatcherAssert.assertThat(
+            dynum.longValue(),
+            Matchers.equalTo(number.longValue())
+        );
+    }
+
+    @Test
+    public void testConstructDyNumFromLongCompareToInt() {
+        final Double number = Double.valueOf(2);
+        final DyNum dynum = new DyNum(number);
+        MatcherAssert.assertThat(
+            dynum.intValue(),
+            Matchers.equalTo(number.intValue())
+        );
+    }
+
+    @Test
+    public void testConstructDyNumFromLongCompareToFloat() {
+        final Float number = Float.valueOf(2);
+        final DyNum dynum = new DyNum(number);
+        MatcherAssert.assertThat(
+            dynum.floatValue(),
+            Matchers.equalTo(number.floatValue())
+        );
+    }
+
+    @Test
+    public void testConstructDyNumFromDouble() {
+        final Double number = Double.valueOf(2);
```

Comment:
on line 41 you use `final double number = 2.0f;` 
but here you use `final Double number = Double.valueOf(2)` for the same value.
You'd better keep the same consistent style.